CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Added support for Android.
 - Introduced experimental feature flag `bEnableResultTypes`, defaulting false. Flip this to true for Interest queries to only include the set of components required to run. Should give bandwidth savings depending on your game. 
 - Moved Dev Auth settings from runtime settings to editor settings.
+- Added a button to generate the Development Authentication Token inside the Unreal Editor.
```

Comment:
Maybe we want to say where in the editor here. Maybe we just leave it to the docs?